app/index.html

```diff
@@ -105,6 +104,22 @@ <h2 class="hit-name">{{{ _highlightResult.name.value }}}</h2>
     </div>
   </script>
 
+  <!-- No-Results template -->
+  <script type="text/template" id="no-results-template">
+    <div id="no-results-message">
+    <p>We didn&#39;t find any results for the search <em>"{{ query }}"</em>.</p>
```

Comment:
It was for syntax highlighting, but I agree that it is more readable with a ' rather than a &#39;. :)